PhotoKingdomAPI/PhotoKingdomAPI/Controllers/Manager.cs

```diff
@@ -382,6 +443,22 @@ public int loadData()
                     {
                         AttractionId = casaloma.Id
                     });
+                    var photowar3 = ds.AttractionPhotowars.Add(new AttractionPhotowar
+                    {
+                        AttractionId = boyerwoodlot.Id
+                    });

```

Comment:
Since these photowars are done (because you already chose winners), should you instantiate an old start date and end date, like the "photowarExpired" object above? If you put expired date for these, and remove the "IsWinner = 1" for the photouploads below, then when we run the api/CheckOwns method, it will determine the winners from the votes, for all expired photowars.

Reply:
same applies for the other photowars below